### ISSUE TITLE
Execute of template failed: template: partials/render/accessibility.html:254:105

### ISSUE THREAD

**Issue by yaaax** [DISIC/frago#32]
hugo version: **v0.92.0-B3549403**
frago version: **v1.6.11**

When starting a project,
`hugo serve` fails if:

- csv file is not empty
- and csv file does not contain at least 1 `na` 1 `nc` and 1 `c`

For example, when:

```
- 1.1 : c  c  c  c  c
- 1.2 : nc nc nc nc nc
```
(no `na`)

Error:
```
Execute of template failed: template: partials/render/accessibility.html:254:105: executing "partials/render/accessibility.html" at <len ($data.Get (printf "%s%s" $scorekey "critNA"))>: error calling len: reflect: call of reflect.Value.Type on zero Value
```
